frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -358,10 +358,7 @@ function DashboardInner(props: DashboardProps) {
       setHasScroll(event.target.scrollTop > 0);
     };
 
-    node.addEventListener("scroll", handleScroll, {
-      capture: false,
```

Comment:
`capture: false` [is a default value ](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#capture). Having it here feels like the default value is otherwise.